NEWS.md

```diff
@@ -1,3 +1,32 @@
+### v9.9.0 (2023-02-06)
+* Added support for url obfuscation using regex (Thanks for your contribution @matewilk)
+  * For Distributed Tracing, this means that:
+     * Incoming and outgoing requests' `path` will be obfuscated
+     * Transaction's `request.url` attribute will be obfuscated
+     * Span's `http.uri` path will be obfuscated
+  * For transactions, this means that:
+    * Transaction trace details `url` will be obfuscated
+  * With the following example configuration, url obfuscation will turn `/api/v1/users/12345456/edit` to `/api/v1/users/**/edit`.
+    ```js
+    url_obfuscation: {
+      enabled: true,
+      regex: {
+        pattern: /(\/api\/v1\/users\/)([\d]+)(\/.*$)/,
+        flags: "i",
+        replacement: '$1**$3'
+      }
+    }
+    ```
+  * You can also use environment variables to configure url obfuscation:
+    ```yml
+    NEW_RELIC_URL_OBFUSCATION_ENABLED: "true",
+    NEW_RELIC_URL_OBFUSCATION_REGEX_PATTERN: '/(\/api\/v1\/users\/)([\d]+)(\/.*$)/',
+    NEW_RELIC_URL_OBFUSCATION_REGEX_FLAGS: 'i',
+    NEW_RELIC_URL_OBFUSCATION_REGEX_REPLACEMENT: '$1**$3'
+    ```
+

```

Comment:
I'm not entirely sure about having this how-to in the release notes. I guess it's fine, but let's not forget to put it in docs-website too.

Reply:
Matt already put one together: https://github.com/newrelic/docs-website/pull/11390

Reply:
Well, it would have been a little embarrassing if we had had to point to the release notes in case someone asked how to obfuscate urls.